wpilib/wpilib/iterativerobotbase.py

```diff
@@ -60,8 +61,14 @@ class Mode(enum.Enum):
 
     # ----------- Overridable initialization code -----------------
 
-    def __init__(self):
+    def __init__(self, period):
+        """Constructor for IterativeRobotBase.
+
+        :param period: Period in seconds
+        """
         self.last_mode = self.Mode.kNone
+        self.period = period
+        self.watchdog = Watchdog(period, self.printLoopOverrunMessage)
         super().__init__()
```

Comment:
Hm. I wonder why this super call is at the end rather than the beginning. Realistically it should be before anything creates any other WPILib objects. Mind moving this?